tidb_test.go

```diff
@@ -463,52 +463,36 @@ func (s *testSessionSuite) TestAutoincrementID(c *C) {
 	mustExecSQL(c, se, s.dropDBSQL)
 }
 
+func checkAutocommit(c *C, se Session, stmt string, isNil, expect bool) {
```

Comment:
Here the expect value is compared with `DisableAutocommit`, so I think it's better to rename this method to `checkDisableAutocommit`.